components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -97,6 +97,11 @@ def innerSequenceType(type):
     return type.inner.inner if type.nullable() else type.inner
 
 
+def innerMozMapType(type):
+    assert type.isMozMap()
+    return type.inner.inner if type.nullable() else type.inner
```

Comment:
We might as well merge this with the innerSequenceType function (innerMapOrSequenceType?), since they're identical except for the asserts.